### ISSUE TITLE
APP_URL not working, I can set whatever i want. It will be ignored

### ISSUE THREAD

**Issue by mschwagerbln** [snipe/snipe-it#6849]
- [ ] I have enabled debug mode 
- [x] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**
APP_URL not working in .env

**To Reproduce**
1. Go to /.env
2. change APP_URL from _https://domain.local/snipeit/public_ to  **https://snipe.domain.de/snipeit/public**
3. php artisan config:cache
4. php artisan config:clear

**Expected behavior**
it still shows to the old local domain (domain.local) instead of snipe.domain.de

**Screenshots**

**Server (please complete the following information):**
 - Snipe-IT Version **last version** 
 - OS: [e.g. Ubuntu, CentOS] **CentOS**
 - Web Server: [e.g. Apache, IIS] **Apache behind nginx-reverse-proxy**
 - PHP Version 7

**Comment by mschwagerbln** [comment]
This is very frustating, because i can set whatever i want, nothing changes (always the old domain name > domain.local instead of snipe.domain.de. This is the first time i had problems like this (only change the domain-name for "reverse proxying") ownCloud = no Problem (out of the box), osTicket = no Problem (out of the box), etc. SnipeIT (very great tool) > changing the domain name in APP_URL > it does not interest > all other variables like mailing work! **APP_URL = NOT!**

**Comment by sjelle** [comment]
Hello,

Did you try php artisan view:clear ?

**Comment by snipe** [comment]
This isn't really possible. The app uses that `APP_URL` and nothing else to determine URLs. 

Please confirm that you're editing the correct .env (the .env in your snipe-it root directory) and not example.env or anything else. 

Also confirm that you've run the commands at the [Common Issues](https://snipe-it.readme.io/docs/common-issues) link:

```
composer dump-autoload
composer clear-compiled
php artisan config:clear
php artisan view:clear
php artisan cache:clear
php artisan route:clear
```

**Comment by PetzJohannes** [comment]
Do you have some rewrite rules in your .htaccess, Apache or nginx configuration?

**Comment by mschwagerbln** [comment]
The Problem was in front of the monitor >> Me! :) Misconfiguration in Reverse Proxy. Sorry, the issue can be closed! (On 4.6.15 found a bug in logout-process > Success Message just showing „auth/...“)
